### DISCUSSION TITLE
Zombie Mod & Maps

### DISCUSSION THREAD

**Discussion by dc-redeclipse** [redeclipse/base#1206] [Maps and Content Showcase]
**Red Eclipse Zombie Mod:**
The Zombie Mode is designed like the most Zombie Mods.
Player fights against a bunch of Zombies.
The Zombies can kill the Player with one hit and had only claws and moves slower as the player.
The Player can use all weapons and had endless ammo. 
All these vars are stored in the "zombiemode.cfg".

The Zombie Mod requires the following mode & mutator settings:
Deathmatch - coop survivor classic hard

The map is designed for the mod:
A small path leads the player from a start point along some grenade and mine spawns along the track. The Zombies need a moment before they get to the player start point and follow them.

_Maps:_
**Dawnwoods**
![dawnwoods](https://user-images.githubusercontent.com/60038236/84075940-388b4180-a9d5-11ea-82e7-9bf626d252a1.png)
by Jonathan 'W!CK3D' Roels, modified by 'dc'

**Small Rats**
![smallrats](https://user-images.githubusercontent.com/60038236/84075954-3f19b900-a9d5-11ea-873b-58db9f2f39a1.png)
by 'SniperGoth', 'Dziq', modified by 'dc'

Maps can make like this for the mod.
All maps and the mod cfg is stored in the zip file.

_Current Version_
v0.2
[Zombie_Mod_Maps_v0.2.zip](https://github.com/redeclipse/world/files/4798649/Zombie_Mod_Maps_v0.2.zip)
zombiemode.cfg for Red Eclipse 2.0.1
[Zombie_Mod_cfg_re2.0.1_v0.2.zip](https://github.com/redeclipse/world/files/4798736/Zombie_Mod_cfg_re2.0.1_v0.2.zip)
zombiemode.cfg for Red Eclipse 2.0.0
[Zombie_Mod_cfg_re2.0.0_v0.2.zip](https://github.com/redeclipse/world/files/4798730/Zombie_Mod_cfg_re2.0.0_v0.2.zip)

Have fun with it!
I am always happy about feedback!

**Comment by dc-redeclipse** [comment]
**Updated to v0.2**
- fix zombie spawn on both maps
- little texture fixes
- fix death materials on smallrats

**Comment by singhamitoj** [comment]
Where do you put the zombiemod.cfg? I tried putting it in multiple places but it would not work. I am using the tarball for Linux on Ubuntu 20.10. Also, how do you make maps for the zombie mod?

Never mind, I figured out that I had to load it.

**Comment by singhamitoj** [comment]
How do you load the zombiemode.cfg? I am not using a server for this. Will this work, or will I need a server for it?

**Reply by dc-redeclipse** [reply]
sorry for the late reply.
just exec zombiemode.cfg in your servinit.cfg
